pkg/concurrency/keyed_rw_mutex_test.go

```diff
@@ -25,7 +25,7 @@ func TestKeyedRWMutexSameKey(t *testing.T) {
 	}()
 
 	a.False(signal.IsDone())
-	km.Unlock(testKey)
+	func() { defer km.Unlock(testKey) }() // suppress the error for calling Unlock()
```

Comment:
```suggestion
	func() { defer km.Unlock(testKey) }() // Suppress linter error for calling Unlock()
```

Reply:
I think we can remove keyed_rw_mutext with it's tests

Reply:
- https://github.com/stackrox/stackrox/pull/9170